hash/nes.xml

```diff
@@ -36561,7 +37119,7 @@ license:CC0
 		<description>Super Spike V'Ball (USA)</description>
 		<year>1990</year>
 		<publisher>Nintendo</publisher>
-		<info name="serial" value="NES-VJ-USA"/>
+		<info name="serial" value="NES-VJ-(USA/CAN)"/>
```

Comment:
Please don’t use this kind of pseudo-regex syntax – just add multiple `info` tags.

Reply:
To be fair, this is perfectly in line with the original format I designed for this softlist.
Having multiple serials with confirmed variants makes the xml noisier, IMHO

I agree with other comments, though

Reply:
Yes, split it into multiple lines like that.  It’s unambiguous when it’s done that way.  The rest of the software list should be cleaned up eventually.  Front-ends already have to handle multiple `info` elements with the same `name` attribute.

One example of multiple `info` elements with the same `name` attribute is `cgenie_cass:natomors`: https://github.com/mamedev/mame/blob/mame0231/hash/cgenie_cass.xml#L358 (multiple `author` values).

Another is `psx:koukrose` (file is too big to practically view in GitHub web UI) with multiple `alt_title` values.